Calories.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Sep 23 00:45:03 2020
+
+@author: olumideakinola
+"""
+
+from clarifai.rest import ClarifaiApp, Image as ClImage
+
+import wolframalpha
+
+
+class Calories:
+ 
+    
+    def __init__(self,image):
+        self.image = image
+    
+    def getFoodName(self):
+        self.lst=[]
+        app = ClarifaiApp(api_key='64f97bc1c2e34da9aeaf6103fea866d4')
+        model = app.models.get('food-items-v1.0')
+        image = ClImage(url=self.image)
+        response_data = model.predict([image])
+        concepts = response_data['outputs'][0]["data"]['concepts']
```

Comment:
extract this as a separate method. Maybe `getNameFromResponse` as this logic is to get the name from list.

Reply:
This gives a list of names